src/http/request.ts

```diff
@@ -147,13 +147,33 @@ export class DrashRequest extends Request {
    * @returns A parsed body based on the content type of the request body.
    */
   async #parseBody(): Promise<ParsedBody> {
-    try {
-      const contentType = this.headers.get(
-        "Content-Type",
```

Comment:
No need for checks on lowercase, it returns the same value for Content-Length and content-length (feel free to test it with a quick fetch in the browser or Deno)

Reply:
fixed: 87d964e. i opted to go with checking lowercase instead of uppercase